system/jlib/jstats.cpp

```diff
@@ -775,190 +775,194 @@ class StatisticMeta
+    { WHENFIRSTSTAT(FirstRow), "The time when the first row is processed by an activity"  },
+    { WHENFIRSTSTAT(QueryStarted), "The time when a query started./nDeprecated" }, // Deprecated - use WhenStart
+    { WHENLASTSTAT(QueryFinished), "The time when a query finished./nDeprecated" }, // Deprecated - use WhenFinished
+    { WHENFIRSTSTAT(Created), "The time when an item was created" },
+    { WHENFIRSTSTAT(Compiled), "The time a workunit started being compiled" },
+    { WHENFIRSTSTAT(WorkunitModified), UNUSED },
+    { TIMESTAT(Elapsed), "The elapsed time between starting and finishing\nFor child queries this may be significantly larger than TimeTotalExecute" },
+    { TIMESTAT(LocalExecute), "The time spent executing this activity not including its inputs\nSort activities by local execute time to help isolate potential processing bottlenecks" },
+    { TIMESTAT(TotalExecute), "The time spent executing this activity and its inputs\nSort activities by total execute time to find sections of a query that are a bottleneck" },
+    { TIMESTAT(Remaining), UNUSED },
+    { SIZESTAT(GeneratedCpp), "The size of the generated c++ file" },
+    { SIZESTAT(PeakMemory), "The peak memory used while processing this item" },
+    { SIZESTAT(MaxRowSize), "The high water mark of the memory used for representing rows (roxiemem)" },
+    { NUMSTAT(RowsProcessed), "The number of rows processed" },
+    { NUMSTAT(Slaves), "The number of parallel execution processes used to execute an activity" },
+    { NUMSTAT(Starts), "The number of times the activity has started executing\nAn activity is active if this does not match NumStops" },
+    { NUMSTAT(Stops), "The number of times the activity has stopped executing\nAn activity is active if this is less than NumStarts" },
+    { NUMSTAT(IndexSeeks), "The number of keyed lookups on an index\nThese correspond to KEYED() filters on indexes.  A single keyed filter may result in multiple seeks if a leading component is not single-valued" },
+    { NUMSTAT(IndexScans), "The number of index scans\nHow many entries are sequentially examined after an initial seek (including wild seeks).  Large numbers compared to the number of seeks may indicate extra keyed filters would be worthwhile" },
+    { NUMSTAT(IndexWildSeeks), "The number of seeks caused by WILD() filters\nThe number of keyed lookups that had to search for the next potential match.  If this is a high proportion of NumIndexScans it may suggest poor key design" },
+    { NUMSTAT(IndexSkips), "The number of smart-stepping operations that increment the next match" },
+    { NUMSTAT(IndexNullSkips), "The number of smart-stepping operations that had no effect\nIf this is large compare to NumIndexSkips it suggests the priority may not be set correctly" },
+    { NUMSTAT(IndexMerges), "The number of merges set up when smart stepping"},
+    { NUMSTAT(IndexMergeCompares), "The number of merge comparisons when smart stepping" },
```

Comment:
Consistency: other instances of smart-stepping are hyphenated

Reply:
I think it is correct as it is.  Using a hyphen because it is multiple words used as an adjective, but here it is not.